controllers/nodehealthcheck_controller.go

```diff
@@ -459,13 +472,22 @@ func (r *NodeHealthCheckReconciler) remediate(node *v1.Node, nhc *remediationv1a
 	}
 
 	// create remediation CR
-	created, err := rm.CreateRemediationCR(remediationCR, nhc)
+	created, leaseRequeueTimeout, err := rm.CreateRemediationCR(remediationCR, nhc)
```

Comment:
>I think here it still is the leaseDuration

I guess it's kind of an egg an chicken dilemma, ATM I'm taking the approach that requeue time is managed in lease manager (vs in `Reconcile` method),  so this code reflects the idea that when a CR is created a requeue time is essential in order to manage that CR's lease (i.e extend it or remove it).

>requeueIn := leaseDuration (-someBuffer!?) )

Basically here it's the same concept as in the above comment, In the approach that I'm currently taking this part (i.e applying the buffer) is already managed in the lease manager so it would be transparent to the reconciler.

>(timeout?)

I've removed `timeout` suffix - I agree it's redundant here.